### PR TITLE
feat: convert `feature_flag` to `bigint` [CW-2767]

### PR DESCRIPTION
**Pull request by scmmishra**
Adding new feature flag breaks the puny 4bytes limit for the `integer` field, this PR converts it to a `bigint` instead.

### CODE REVIEW

**Comment by scmmishra** [comment]
@pranavrajs @sojan-official please review whenever you can

File: db/schema.rb [hunk 1]
```diff
@@ -617,6 +619,19 @@
...
+    t.string "labelable_type", null: false
+    t.bigint "labelable_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["label_id"], name: "index_label_associations_on_label_id"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_contact", where: "((labelable_type)::text = 'Contact'::text)"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_conversation", where: "((labelable_type)::text = 'Conversation'::text)"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_labelable"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_labelable_type_and_labelable_id"
+  end
```
Comment (sojan-official) [inline db/schema.rb:633]: @scmmishra  this might not be intended

File: db/schema.rb [hunk 2]
```diff
@@ -946,6 +961,7 @@
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
   add_foreign_key "inboxes", "portals"
+  add_foreign_key "label_associations", "labels"
```
Comment (sojan-official) [inline db/schema.rb:964]: @scmmishra  this might not be intended

File: db/schema.rb [hunk 3]
```diff
@@ -451,8 +451,10 @@
     t.datetime "assignee_last_seen_at", precision: nil
     t.datetime "first_reply_created_at", precision: nil
     t.integer "priority"
-    t.bigint "sla_policy_id"
     t.datetime "waiting_since"
+    t.bigint "sla_policy_id"
+    t.text "summary"
+    t.datetime "summary_generated_at"
```
Comment (sojan-official) [inline db/schema.rb:457]: @scmmishra  this might not be intended

**Review by sojan-official — COMMENTED** [review]
change looks good to me. please make sure to clean up the schema file based on the comments

**Review by sojan-official — APPROVED** [review]

**Pull request merged**